core/arch/arm/plat-rcar/main.c

```diff
@@ -53,7 +53,7 @@ register_dynamic_shm(NSEC_DDR_3_BASE, NSEC_DDR_3_SIZE);
 
 static void main_fiq(void);
 
-static const struct thread_handlers handlers = {
+static const struct thread_handlers handlers __nex_data = {
```

Comment:
Isn't this needed for `plat-vexpress`, too? (I think you have tried virtualization with QEMUv8, right?).

Reply:
It is more complex than this. Yes, I initially did this on QEMUv8, but I had to enable gicv3.
Problem is that QEMU does not support virtualization extensions for gicv2, only for gicv3. So I had to rebuild both ARM-TF and OP-TEE with gicv3 enabled. 

This is why I didn't upstreamed changes to `plat-vexpress`. It requires multiple simultaneous changes in OP-TEE, ARM-TF and in build scripts. Also, I had to backport some virtualization-related patches to that version of QEMU that OP-TEE build scripts use.